config/environments/production.rb

```diff
@@ -60,6 +60,7 @@
 
   # Use a different cache store in production.
   # config.cache_store = :mem_cache_store
+  config.cache_store = :memory_store, { size: 10.megabytes }
```

Comment:
Was the 10MB size chosen by any kind of scientific process or just a guess (which is totally OK)?